src/python_testing/TC_OpstateCommon.py

```diff
@@ -0,0 +1,1191 @@
+                 TestStep(7, "TH reads from the DUT the GeneratedCommandList attribute")
+                 ]
+        return steps
+
+    async def TEST_TC_OPSTATE_BASE_1_1(self, endpoint=1, cluster_revision=1, feature_map=0):
+        cluster = self.test_info.cluster
+        attributes = cluster.Attributes
+        events = cluster.Events
+        commands = cluster.Commands
+
+        self.init_test()
+
+        # STEP 1: Commission DUT to TH (can be skipped if done in a preceding test)
+        self.step(1)
+
+        # STEP 2: TH reads from the DUT the ClusterRevision attribute
+        self.step(2)
+        await self.read_and_expect_value(endpoint=endpoint,
+                                         attribute=attributes.ClusterRevision,
+                                         expected_value=cluster_revision)
+
+        # STEP 3: TH reads from the DUT the FeatureMap attribute
+        self.step(3)
+        await self.read_and_expect_value(endpoint=endpoint,
+                                         attribute=attributes.FeatureMap,
+                                         expected_value=feature_map)
+
+        # STEP 4: TH reads from the DUT the AttributeList attribute
+        self.step(4)
+        expected_value = [
+            attributes.PhaseList.attribute_id,
+            attributes.CurrentPhase.attribute_id,
+            attributes.OperationalStateList.attribute_id,
+            attributes.OperationalState.attribute_id,
+            attributes.OperationalError.attribute_id,
+            attributes.GeneratedCommandList.attribute_id,
+            attributes.AcceptedCommandList.attribute_id,
+            attributes.AttributeList.attribute_id,
+            attributes.FeatureMap.attribute_id,
+            attributes.ClusterRevision.attribute_id
+        ]
+
+        if self.check_pics(("%s.S.A0002" % self.test_info.pics_code)):
+            expected_value.append(attributes.CountdownTime.attribute_id)
+
+        await self.read_and_expect_array_contains(endpoint=endpoint,
+                                                  attribute=attributes.AttributeList,
+                                                  expected_contains=expected_value)
+
+        # STEP 5: TH reads from the DUT the EventList attribute
+        self.step(5)
+        if self.pics_guard(self.check_pics("PICS_EVENT_LIST_ENABLED")):
+            expected_value = [
+                events.OperationalError.event_id,
+            ]
+
+            if self.check_pics(("%s.S.E01" % self.test_info.pics_code)):
+                expected_value.append(events.OperationCompletion.event_id)
+
+            await self.read_and_expect_array_contains(endpoint=endpoint,
+                                                      attribute=attributes.EventList,
+                                                      expected_contains=expected_value)
+
+        # STEP 6: TH reads from the DUT the AcceptedCommandList attribute
```

Comment:
All of these test steps in the 1.1 are not required --> the conformance test covers all of it for all tests, without requiring to have this duplication.

Reply:
I'm gonna keep it here once this is described in the test plan, so we keep the consistency. What you think ?